packages/api/src/hooks/useAccountsList.ts

```diff
@@ -32,6 +32,8 @@ const useAccountsList = () => {
                 loginid: `${account.loginid}`,
                 /** Account's currency config information */
                 currency_config: account.currency ? getConfig(account.currency) : undefined,
+                /** Determines the account type whether it is a real or demo account. */
+                account_type: account.is_virtual ? 'demo' : 'real',
```

Comment:
Why do we need `account_type`? We can always refer to the is_virtual flag to know if the account is real or demo.